src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -36,6 +36,33 @@ const navIconProps = {
   hover: { opacity: 1 }, focus: 'hover'
 }
 
+const WorkspacePermissionNotice = ({ workspace }) => {
+  const isReadOnly = !Utils.canWrite(workspace.accessLevel)
+  const isLocked = workspace.workspace.isLocked
+
+  return (isReadOnly || isLocked) && span({
+    style: {
+      display: 'inline-flex',
+      alignItems: 'center',
+      height: '2rem',
+      padding: '0 1rem',
+      borderRadius: '1rem',
+      marginRight: '2rem',
+      backgroundColor: colors.dark(0.15),
+      textTransform: 'none'
+    }
+  }, [
+    isLocked ? icon('lock', { size: 16 }) : icon('eye', { size: 20 }),
```

Comment:
With these icons the same size, either the eye looked too small or the lock looked too big.